README.md

```diff
@@ -150,6 +150,21 @@ packager URL directly, first add a Chrome extension to send an
 `AMP-Cache-Transform: any` request header. Otherwise, follow the above
 "Demonstrate privacy-preserving prefetch" instructions.
 
+#### Testing productionization without a valid certificate
+
+It is possible to test an otherwise fully production configuration without
+obtaining a certificate with the `CanSignHttpExchanges` extension. `amppkg`
+still needs to perform OCSP verification, so the Root CA must be valid (i.e. no
```

Comment:
s/Root/Issuer/ would be more accurate, I think.

Reply:
Resolved in commit https://github.com/ampproject/amppackager/pull/278/commits/87ce94ae49b53beb76a96eff6b02e94ab515534d